src/snowflake/cli/plugins/cortex/commands.py

```diff
@@ -1,6 +1,60 @@
+from __future__ import annotations
+
+from typing import List, Optional
+
+import typer
+from snowflake.cli.api.cli_global_context import cli_context
 from snowflake.cli.api.commands.snow_typer import SnowTyper
+from snowflake.cli.api.output.types import CollectionResult
 
 app = SnowTyper(
     name="cortex",
     help="Provides access to Snowflake Cortex LLM.",
 )
+
+
+@app.command(
+    requires_connection=True,
+)
+def search(
+    query: str = typer.Argument(help="Query to look for in your data"),
+    service: str = typer.Option(
+        help="Cortex search service to be used. Example: --service my_cortex_service",
+    ),
+    columns: Optional[List[str]] = typer.Option(
+        help='Columns that will be returned with the results. If none is provided, only search column will be included in results. Example --columns "foo" --columns "bar"',
```

Comment:
The SQL support comma separated format, I think we should do the same 🤔

Reply:
This seems to be the option preffered by Typer